src/highlighting.c

```diff
@@ -1159,6 +1159,30 @@ void highlighting_set_styles(ScintillaObject *sci, GeanyFiletype *ft)
 }
 
 
+/** Retrieves a @a named_style.
+ * If a style with the name @a named_style does not exist, NULL will be returned.
+ * @param named_style Key/name of the style as specified in the colorscheme file
+ * @return A pointer to the style struct.
+ */
+GEANY_API_SYMBOL
+const GeanyLexerStyle *highlighting_get_named_style(const gchar *named_style)
+{
+	GeanyLexerStyle *cs;
+	gchar *comma, *name = NULL;
+
+	g_return_val_if_fail(named_style, NULL);
+	name = utils_strdupa(named_style);	/* named_style must not be written to, may be a static string */
+
+	comma = strstr(name, ",");
+	if (comma)
+	{
+		*comma = '\0';	/* terminate name to make lookup work */
+	}
```

Comment:
Whats this doing? If you pass a name why does it need to terminate at a comma?  Either this is functionality specific to the particular useage, in which case it should be in the caller, or it should be documented behaviour with an explanation of why.

Reply:
I removed it. It was a leftover copied in from ```read_named_style()```. But it doesn't make sense any more.